core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -110,9 +112,15 @@ abstract contract FeePayer is Testable {
         );
         lastPaymentTime = time;
 
+        totalPaid = regularFee.add(latePenalty);
+        if (totalPaid.isEqual(0)) {
+            return totalPaid;
+        }
+        // The amount of fees paid must be < pfc or the fee will be larger than 100%.
+        require(_pfc.isGreaterThan(totalPaid));
```

Comment:
I could totally be missing something, but it's not clear to me that reverting is the right thing to do here.

Wouldn't that just permanently lock the collateral in the contract? Are there other ways to mitigate this issue? 

Also, won't it already revert if this is > 100%? So isn't this the same behavior we already have?

Reply:
You could set fees to 0 to unlock the collateral. It will already revert, but it will be a SafeMath-underflow error and I prefer failing earlier than that.

Reply:
Another solution would be to just exit early and not charge fees, or charge a maximum of 100%

Reply:
> Another solution would be to just exit early and not charge fees, or charge a maximum of 100%

Just maxing it at 100% is interesting and at least keeps the capital from being locked. Do you see any disadvantages to doing it this way?

Reply:
Nope I think its a worthwhile addition. I'll add